ci/container-images/permaplant-node/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:slim
+
+RUN mkdir /.npm
+RUN chown 47110:47110 -R /.npm
```

Comment:
I think we will need an `npm install` here in order for the commands from https://github.com/ElektraInitiative/PermaplanT/pull/171 to work.